mk/config.mk

```diff
@@ -176,6 +176,31 @@ CFG_WITH_USER_TA ?= y
 # case you implement your own TA store
 CFG_REE_FS_TA ?= y
 
+# Support for loading user TAs from a special section in the TEE binary.
+# Such TAs are available even before tee-supplicant is available (hence their
+# name), but note that many services exported to TAs may need tee-supplicant,
+# so early use is limited to a subset of the TEE Internal Core API (crypto...)
+# To use this feature, set EARLY_TA_PATHS to the paths to one or more TA ELF
```

Comment:
I have already added `make ta_dev_kit`, after @airbak's suggestion above. So indeed, we could very well use `CFG_EARLY_TA_PATHS` as you suggest and still avoid re-compilation.
Let me give it a try and update ;-)